content/businessapps/gettingStarted.adoc

```diff
@@ -39,20 +39,68 @@ data model classes something more than they are - include too much of implementa
 
 *Business assets project* represents your business logic as processes, cases, rules, decision tables and more.
 
-== Build your first Business Application
+Following are the maven archetype commands need to be used to generate different types of projects:
+
+* Data model archetype
+
+[source, bash,subs="attributes+"]
+----
+mvn archetype:generate -B -DarchetypeGroupId=org.kie -DarchetypeArtifactId=kie-model-archetype -DarchetypeVersion={COMMUNITY_VERSION_FINAL} -DgroupId=com.company -DartifactId=business-application-model -Dversion=1.0-SNAPSHOT -Dpackage=com.company.model
+----
+
+`org.apache.maven.archetypes:maven-archetype-quickstart:1.3`
+
+* Business assets project archetype
+
+[source, bash,subs="attributes+"]
+----
+mvn archetype:generate -B -DarchetypeGroupId=org.kie -DarchetypeArtifactId=kie-kjar-archetype -DarchetypeVersion={COMMUNITY_VERSION_FINAL} -DgroupId=com.company -DartifactId=business-application-kjar -Dversion=1.0-SNAPSHOT -Dpackage=com.company
+----
+`org.kie:kie-kjar-archetype:{COMMUNITY_VERSION_FINAL}`
```

Comment:
This doesn't work as you expect. The variables aren't resolved when you build the website.

Due to a bug in JBake, the only way I managed to make variables work is by converting the adoc content to ftl. You can see how I did it in https://github.com/kiegroup/jbpm-website/blob/main/templates/_content_businessapps_examples.ftl. This template is used by https://github.com/kiegroup/jbpm-website/blob/main/content/businessapps/examples.adoc

You can use https://asciidoclive.com/edit/scratch/1 to convert the adoc code to HTML. You can see the generated HTML by clicking on the gear icon in the top right corner and then click in "Show output HTML".

Reply:
Thank you Helber, made the changes as suggested.